<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>F_Care</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 247);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1285</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionConfigProxy"/>
    <addaction name="actionConfigClone"/>
    <addaction name="actionConfigInteract"/>
    <addaction name="actionView_log"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionToken"/>
    <addaction name="actionViewClone"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <widget class="QMenu" name="menuCh_y">
     <property name="title">
      <string>Chạy</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionRunAllAccount"/>
    </widget>
    <widget class="QMenu" name="menuD_ng">
     <property name="title">
      <string>Dừng</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionStopAllAccount"/>
    </widget>
    <addaction name="menuCh_y"/>
    <addaction name="menuD_ng"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
   </widget>
   <widget class="QMenu" name="menuCheckPoint">
    <property name="title">
     <string>CheckPoint</string>
    </property>
    <addaction name="actionCheckPoint"/>
   </widget>
   <widget class="QMenu" name="menuVersion">
    <property name="title">
     <string>Version</string>
    </property>
    <addaction name="action_F_care_Info"/>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuWindow"/>
   <addaction name="menuCheckPoint"/>
   <addaction name="menuVersion"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionConfigProxy">
   <property name="text">
    <string>Cấu hình thiết lập proxy</string>
   </property>
  </action>
  <action name="actionConfigClone">
   <property name="text">
    <string>Cấu hình Clone</string>
   </property>
  </action>
  <action name="actionRunAllAccount">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/round-play-button.png</normaloff>:/images/round-play-button.png</iconset>
   </property>
   <property name="text">
    <string>Chạy tất cả tài khoản</string>
   </property>
  </action>
  <action name="actionStopAllAccount">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/icons8_stop_1.ico</normaloff>:/images/icons8_stop_1.ico</iconset>
   </property>
   <property name="text">
    <string>Dừng tất cả tài khoản</string>
   </property>
   <property name="toolTip">
    <string notr="true">Dừng tất cả tài khoản</string>
   </property>
  </action>
  <action name="actionConfigInteract">
   <property name="text">
    <string>Cấu hình tương tác</string>
   </property>
  </action>
  <action name="actionImport_account">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/icons8_add_24px.png</normaloff>:/images/icons8_add_24px.png</iconset>
   </property>
   <property name="text">
    <string>Import account</string>
   </property>
  </action>
  <action name="actionUpdate_account">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/icons8_refresh_23px_1.png</normaloff>:/images/icons8_refresh_23px_1.png</iconset>
   </property>
   <property name="text">
    <string>Update account</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionToken">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/images/icons8_settings.png</normaloff>:/images/icons8_settings.png</iconset>
   </property>
   <property name="text">
    <string>Token</string>
   </property>
  </action>
  <action name="actionViewClone">
   <property name="text">
    <string>View clone</string>
   </property>
  </action>
  <action name="actionView_log">
   <property name="text">
    <string>View log</string>
   </property>
  </action>
  <action name="actionCheckPoint">
   <property name="text">
    <string>CheckPoint</string>
   </property>
  </action>
  <action name="action_F_care_Info">
   <property name="text">
    <string>Thông tin F-care</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
